docs/pre-commit.md

```diff
@@ -104,21 +107,33 @@ locate the `- id: pymarkdown` line in the file and change it
 in one of these two ways:
 
 ```yaml
-      - id: pymarkdown
-        args: [option1, scan]
+      -   id: pymarkdown
+          args: [option1, scan]
 ```
 
 or:
 
 ```yaml
-      - id: pymarkdown
```

Comment:
interesting.  I had these spaced the same as the sample above.  are you noticing issues with parsing?

Reply:
I changed the formatting to match the formatting in the YAML file I was working with, but it may matter that the formatting is consistent rather than that the formatting is one way or another. With regard to the formatting my own YAML file, I _think_ it's formatting that's been linted by pre-commit, but I'd have to double-check that.